lib/DraggableCore.es6

```diff
@@ -331,6 +345,10 @@ export default class DraggableCore extends React.Component {
     // We're on a touch device now, so change the event handlers
     dragEventFor = eventsFor.touch;
 
+    // Remove the event listener that stopped document scrolling
+    document.removeEventListener('touchmove', this.removeScroll, false);
```

Comment:
But then when the user is touching the device in order to scroll, won't that stop scrolling when they want to scroll?